modules/Bio/EnsEMBL/Compara/PipeConfig/ProteinTrees_conf.pm

```diff
@@ -3529,10 +3551,7 @@ sub core_pipeline_analyses {
             -parameters => {
                 'taxlevels'             => $self->o('goc_taxlevels'),
             },
-            -flow_into  => {
-                '1->A' => WHEN( 'scalar(@{#taxlevels#})' => 'goc_entry_point' ),
-                'A->1' => ['polyploid_move_back_factory'],
-            },
+            -flow_into  => WHEN( 'scalar(@{#taxlevels#})' => 'goc_entry_point' ),
```

Comment:
```suggestion
            -flow_into  => WHEN('scalar(@{#taxlevels#})' => 'goc_entry_point'),
```
(just because you have done it with the previous ones :) )